lib/player.rb

```diff
@@ -0,0 +1,11 @@
+class Player
+  attr_accessor :name, :game, :sym
```

Comment:
- [ ]  `attr_accessor` isn't needed here, and you should be fine with `attr_reader`/`attr_writer`, please change it.